pom.xml

```diff
@@ -542,7 +541,7 @@
                                 <goal>npm</goal>
                             </goals>
                             <configuration>
-                                <arguments>install --ignore-scripts --registry=http://artifacts.codice.org/content/repositories/npm-registry/</arguments>
+                                <arguments>install --ignore-scripts</arguments>
```

Comment:
To make sure releases are reproducible and to maintain security guidelines, please shrink wrap all of the current versions for this PR.  After shrink wrap is added, I think it will be safe to remove the Codice NPM registry.